src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,370 @@
+We employ an approach that we call "Defensive Moderation" -- an emerging concept that focuses primarily on harm 
+reduction and investigative moderation, keeping an eye on behaviours and trends in the wider community and acting upon
+them to prevent issues before they happen.
+
+The reason our approach feels so different to other communities' is because we investigate and act upon issues that
+appear in the wider community, even if they don't directly affect Quilt. We believe that nothing happens in a vacuum,
+and a problematic situation somewhere else is good evidence that the people involve may cause similar situations in our
+spaces. We're not shy about banning problem users before they even set foot in Quilt community spaces, and some people
+find this perplexing -- but we believe in this approach, and it's been a big contributor in keeping Quilt users safe
+from discrimination and other issues.
+
+Similarly, Quilt itself doesn't operate in a vacuum - we operate a program known as Quilt Community Collab, where we
+work with other related communities on moderation-related issues and events. Working alongside organisations such as
+Forge, Geyser and Paper means that we can contribute to improving the _entire_ community sphere, for everyone -- and
+in return, those organisations help us with our investigations and moderation-related issues.
+
+We believe that all communities can contribute positively to the well-being of their users, and that they have a
+responsibility to create safe, inclusive spaces, and to set a trend doing so -- encouraging more communities to do the
+same, and making the Internet a better place for everyone.
+
+{% enddetails %}
+{% endadmonition %}
+
+{% admonition %}
+{% details %}
+{% summary %}Does Quilt send my mods to the servers I play on?{% endsummary %}
+
+This is a misconception based on poor communication: at one point, to help ease the moderation burden in Quilt's
+community spaces, we were thinking about potentially sending the client's list of mods to the servers they connect to
+(like what Forge does). However, this was never formalized -- no RFC was made, and nothing like this was
+implemented for the beta.
+
+Since then, better ideas have appeared. One example is in the in-progress 
+[RFC #17](https://github.com/QuiltMC/rfcs/pull/17), which proposes a standard feature toggle system -- allowing servers
+to tell clients what they expect of them, and for mods to configure themselves based on those expectations.
+
+We prefer a solution that doesn't impact the privacy of our users -- we're still looking at alternatives, but we do
+believe that the above RFC may be the correct solution.
+
+{% enddetails %}
+{% endadmonition %}
+
+{% admonition %}
+{% details %}
+{% summary %}Why don't you explain the Fabric drama here?{% endsummary %}
+
+This is a question we've been getting a lot since Quilt entered beta -- it's something that sounds reasonable on paper, 
```

Comment:
```suggestion
This is a question we've been getting a lot since Quilt entered beta — it's something that sounds reasonable on paper, 
```
I've decided to stop making suggestions for em dashes, just do find and replace or something, I suppose?